BattleShipsClient/BattleShipsClient/Program.cs

```diff
@@ -11,12 +12,42 @@ static class Program
         /// <summary>
         /// The main entry point for the application.
         /// </summary>
+        
         [STAThread]
         static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Form1());
+            Application.Run(new Menu());
+            //Application.Run(new Form1());
+        }
+        static string LogName;
+        static object LogLock = new object();
+        public static void Log(string message)
+        {
+            lock (LogLock)
+            {
+                using (StreamWriter swAppend = File.AppendText(LogName))
+                {
+                    swAppend.WriteLine($"[{DateTime.Now.Hour.ToString()}:{DateTime.Now.Minute.ToString()}:{DateTime.Now.Second.ToString()}] - {message}");
+                }
+            }
+        }
+        public static void MakeLog()
+        {
+            if (!Directory.Exists("Logs/"))
+            {
+                Directory.CreateDirectory("Logs");
+            }
+            LogName = $"logs/ {DateTime.Today.Day.ToString()}-{DateTime.Today.Month.ToString()}&{DateTime.Now.Hour.ToString()};{DateTime.Now.Minute.ToString()}.txt";
```

Comment:
Is it entirely nessecary to use `-`, `&` and `;` as seperators?
Why not use `-` for all three? 
- Its easier to read
- It doesnt look like something broke
- If you need to get date from name: `fileName.split('-'); day = split[0]; month = day[1]; year = day[2];`